docs/proposals/mvp-dashboard-design.md

```diff
@@ -10,16 +10,37 @@ Last updated: 2017-01-12
 
 ## Abstract
 
-A short summary of the proposal.
+We propose a dashboard/server that allows for the automated retrieval,
+archive, format, and display of performance tests run in the performance lab.
+This tool will allow our team to investigate performance issues and facilitate
+the sharing of lab results with other teams.
```

Comment:
The system should actually retrieve, and display performance test results from any lab. At the end of the day is just another filter in the query. 

What do you mean by 'format'?

Instead of 'performance test runs', we should say 'performance test results'.

I don't think the tool well help us in investigating issues, but instead actually find them. Track the performance over time across builds and different environment and configurations, doing so we will be able to spot regressions. I agree with sharing the results, but it's not just for the performance lab.